pages/notifications.tsx

```diff
@@ -66,11 +77,12 @@ export default function Notification(): JSX.Element {
     },
   ];
   const handleBack = () => {
-    console.log('Click back');
+    router.back();
   };
   const handleNext = () => {
-    console.log('Click next');
-    console.log(JSON.stringify(values, null, 2));
+    const notification = transformToNotificationData(values);
+    handleSetNotifications(notification ? [notification] : []);
+    router.push('/advanced');
```

Comment:
Clicking next should go to /download

Reply:
Changed. Please check again @nicnocquee.